indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import random
+
+from flask import request
+from werkzeug.exceptions import BadRequest
+
+from indico.core.db.sqlalchemy.colors import ColorTuple
+from indico.modules.events.sessions.controllers.management import RHManageSessionsBase, RHManageSessionBase
+from indico.modules.events.sessions.forms import SessionForm
+from indico.modules.events.sessions.models.sessions import Session
+from indico.modules.events.sessions.operations import create_session, update_session, delete_session
+from indico.modules.events.sessions.util import get_colors, get_active_sessions
+from indico.modules.events.sessions.views import WPManageSessions
+from indico.web.flask.templating import get_template_module
+from indico.web.forms.base import FormDefaults
+from indico.web.util import jsonify_data, jsonify_template
+
+
+def _render_session_list(event):
+    tpl = get_template_module('events/sessions/management/_session_list.html')
+    return tpl.render_session_list(event, get_active_sessions(event), get_colors())
```

Comment:
I think you get the args for this twice, once here and once in the kwargs in `RHSessionsList`.

Maybe a `_get_session_list_args` helper function would be a good idea? In this case it's pretty straightforward (the contributions modules has `_get_contribution_list_args` but it's a little bit more complex) so I guess it makes no real difference. But in case we need to add more things in the future...